armory/data/adversarial/resisc45_densenet121_univpatch_and_univperturbation_adversarial_224x224.py

```diff
@@ -107,7 +108,13 @@ def _info(self):
                             shape=[224, 224, 3], dtype=tf.uint8, encoding_format="png"
                         ),
                     },
-                    "label": tfds.features.ClassLabel(names=_LABELS),
+                    "labels": {
```

Comment:
The key here was changed from "label" to "labels". This requires a change below.